libs/search-widget/src/core/models.ts

```diff
@@ -9,16 +9,29 @@ import type {
   WidgetFeatures,
 } from '@nuclia/core';
 
-export const NO_RESULTS: Search.FindResults = {
+export interface FindResultsAsList extends Omit<Search.FindResults, 'resources'> {
+  resultList: Search.FieldResult[];
+}
+
+const EMPTY_FIND_RESULTS: Omit<Search.FindResults, 'resources'> = {
   type: 'findResults',
-  resources: {} as { [id: string]: Search.FindResource },
   total: 0,
   page_number: 0,
   next_page: false,
   page_size: 0,
   query: '',
 };
 
+export const NO_RESULTS: Search.FindResults = {
```

Comment:
not needed anymore

Reply:
it is used in Suggestion. That reminds me I forgot to test it